src/components/Toggle/Slide.tsx

```diff
@@ -23,7 +23,10 @@ const Slide = styled.div<SlideProps>`
     height: 1rem;
 
     cursor: ${props => (props.disabled ? 'not-allowed' : 'pointer')};
-    background: ${props => (props.disabled ? Colors.AUTHENTIC_BLUE_50 : Colors.AUTHENTIC_BLUE_200)};
+    background: ${props =>
+        props.disabled
+            ? SemanticColors.background.secondary
+            : SemanticColors.border.primary}; // TODO should this come from background
```

Comment:
I think that it would make more sense for this to be `SemanticColors.background.{something}`, but we don't have this color (AUTHENTIC_BLUE_200) there at the moment

Reply:
Can you sync with someone in the design team? Pietro, Vini or Fernanda?

Reply:
Hey @nlopin, sorry for taking so long, I checked the [Figma file](https://www.figma.com/file/dTefUItBx5bfbxcCcMzMXm/Wave-DS?node-id=325%3A2) for the Toggle and I see that the colors (with semantic names) are included but we are not following the same convention in the code.
In the Figma there is a `colors.forms.toggle.default.frameOff` but we don't even have the `forms` category.

<img width="368" alt="image" src="https://user-images.githubusercontent.com/46452321/190094775-c55b6957-205c-40fd-99e7-88ed6e07e80e.png">

Should we be aligned so that the tokens in Figma and our colors structure are the same? I believe so but I want to double check before starting a conversation with designers about this.

Reply:
I think we should. Let's find who can we sync with. Ideally we can ask designers to export their tokens from Figma and use them

Reply:
I've asked them about it, [their colors](https://www.figma.com/file/dTefUItBx5bfbxcCcMzMXm/Wave-DS?node-id=137%3A0) already look a lot like ours, so I think it's just a matter of adding the ones that we are missing in the code. I've created #286 to take care of that.